src/components/ui/command-copier/command-copier.scss

```diff
@@ -0,0 +1,59 @@
+@import '../../../styles/variables';
+
+.container {
+  position: relative;
+  display: flex;
+}
+
+.run-command-value {
+  display: flex;
+  flex-grow: 1;
+  align-items: center;
+  font-family: $font-monospace;
+  font-size: 0.9em;
+
+  &--no-visible {
+    opacity: 0;
+  }
+}
+
+.copy-message {
+  position: absolute;
+  top: 0;
```

Comment:
It seems this needs to be `top: 2px;` instead. I noticed that the `Copied to clipboard.` text doesn't sit exactly along the same baseline as the code text that's being copied.

Reply:
I've just tried adding your suggested setting but unforunately that wouldn't work - the current text aligns perfectly with how the spacing had laid out for the other fields in the metadata panel as well. ( see below screenshots)

<img width="408" alt="image" src="https://user-images.githubusercontent.com/27775658/154540185-589fbf57-1e42-4449-974a-a5ceabb678d9.png">

<img width="404" alt="image" src="https://user-images.githubusercontent.com/27775658/154540338-af3c8222-2eaa-4319-a71d-f410d574b888.png">

I had a check and the text do align with the `Copied to clipboard` text do align with the same baseline as the code text. I think what you meant here is that the copy command logo here not aligning with the top of the copy button, which I had adjusted to help with this alignment.

Reply:
just saw your message - I see what you meant now. I've put in a fix 😄